src/plugins/discover/public/services/doc_views/components/doc_viewer_source/source.test.tsx

```diff
@@ -111,7 +111,7 @@ describe('Source Viewer component', () => {
         />
       </KibanaContextProvider>
     );
-    expect(comp.children()).toMatchSnapshot();
+    expect(comp.children().render()).toMatchSnapshot();
```

Comment:
I think it would be better to revert this change and leave a comment in the code that adding `.render()` here would remove Editor components from the snapshot but they are important for the test.

Reply:
Reverted in [7558150](https://github.com/elastic/kibana/pull/132257/commits/7558150a1eacee5ecb159b64e8f015af2bc23d25)

Reply:
Thanks!